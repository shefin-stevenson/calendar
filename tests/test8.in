1
event1#
12/20 12:30
12/21 12:31
blah blah blah keyword blah blah#
1
event2#
12/20 12:20
12/20 12:30
blah blah blah not keywrd blah blah#
4
8
keyword#
9
