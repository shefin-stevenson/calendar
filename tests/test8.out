

Enter a number from the list below to select an action, or press 9 to quit

Edit Commands 

1. Insert new event
2. Delete event
3. Edit event

 Print Commands

4. Print all
5. Print time block
6. Print conflicts

 Search Commands

7. Search by name
8. Search by keyword
Enter new event name (no more than 16 characters):
Enter new event start time (mm/dd hh:mm):
Enter new event end (mm/dd hh:mm):
Enter new event description (no more than 256 characters):

Enter a number from the list below to select an action, or press 9 to quit

Edit Commands 

1. Insert new event
2. Delete event
3. Edit event

 Print Commands

4. Print all
5. Print time block
6. Print conflicts

 Search Commands

7. Search by name
8. Search by keyword
Enter new event name (no more than 16 characters):
Enter new event start time (mm/dd hh:mm):
Enter new event end (mm/dd hh:mm):
Enter new event description (no more than 256 characters):

Enter a number from the list below to select an action, or press 9 to quit

Edit Commands 

1. Insert new event
2. Delete event
3. Edit event

 Print Commands

4. Print all
5. Print time block
6. Print conflicts

 Search Commands

7. Search by name
8. Search by keyword
event2#
12/20 12:20
12/20 12:30
blah blah blah not keywrd blah blah#


event1#
12/20 12:30
12/21 12:31
blah blah blah keyword blah blah#




Enter a number from the list below to select an action, or press 9 to quit

Edit Commands 

1. Insert new event
2. Delete event
3. Edit event

 Print Commands

4. Print all
5. Print time block
6. Print conflicts

 Search Commands

7. Search by name
8. Search by keyword
Enter a keyword to search for:event1#
12/20 12:30
12/21 12:31
blah blah blah keyword blah blah#




Enter a number from the list below to select an action, or press 9 to quit

Edit Commands 

1. Insert new event
2. Delete event
3. Edit event

 Print Commands

4. Print all
5. Print time block
6. Print conflicts

 Search Commands

7. Search by name
8. Search by keyword
